addons/website/data/website_data.xml

```diff
@@ -512,6 +512,11 @@
             <field name="type">url</field>
             <field name="url">/website/static/src/img/snippets_demo/s_references_5.png</field>
         </record>
+
+        <!-- update default seo meta keywords for home page -->
+        <function model="website.page" name="set_website_meta_keywords">
```

Comment:
to remove, no more needed in last spec

And logic is bad :/ call on all pages a method that filter on only one ??

Reply:
@JKE-be Do we need to keep one default entry for keyword(company name entry) or not?

Reply:
no, spec have been updated in the chatter


![image](https://user-images.githubusercontent.com/4848797/69153448-88a8c200-0ade-11ea-89cb-a8a9cc9427f0.png)

Understand Keyword instead of Tag ;)